demo/src/main/java/com/braintreepayments/api/demo/DemoFragment.java

```diff
@@ -14,14 +14,15 @@
 import androidx.fragment.app.Fragment;
 
 import com.braintreepayments.api.BrowserSwitchException;
+import com.braintreepayments.api.BrowserSwitchListener;
 import com.braintreepayments.api.BrowserSwitchOptions;
 import com.braintreepayments.api.BrowserSwitchResult;
 import com.braintreepayments.api.BrowserSwitchStatus;
 
 import org.json.JSONException;
 import org.json.JSONObject;
 
-public class DemoFragment extends Fragment implements View.OnClickListener {
+public class DemoFragment extends Fragment implements View.OnClickListener, BrowserSwitchListener {
```

Comment:
I think we should add `@Override` to the `onBrowserSwitchResult` method for this listener, but I can't comment on that line in this PR (line 104)